gateway/errors.go

```diff
@@ -0,0 +1,18 @@
+package gateway
+
+import (
+	"fmt"
+)
+
+// Error represents the error returned by a handler to the requestor
+type Error struct {
+	Type    string `json:"type"`
```

Comment:
Can we make this a https://github.com/manifoldco/go-manifold/blob/master/errors/errors.go#L18 ?